### PR TITLE
task solution 'moyo logo'

### PR DESCRIPTION
**Pull request by marichkamt**
- [DEMO LINK](https://marichkamt.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://marichkamt.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,62 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-img">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav>">
+        <ul class="nav-list">
+
```
[the remark this answers is not in the document]
Comment (marichkamt) [reply src/index.html:31]: ok,but then i get a linter error.

Screenshot 2023-11-27 at 10 12 55 PM

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,67 @@
...
+    <header class="header">
+      <a href="#" class="logo-img">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav>">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active"
+            href="#"
+            >Apple</a>
```
Comment (anastasiiavorobiova) [inline src/index.html:30]:
```suggestion
            <a 
              class="nav-link is-active"
              href="#"
            >
              Apple
            </a>
```
Consider fixing indents according to the code style guide

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,67 @@
...
+          </li>
+          <li class="nav-item">
+            <a class="nav-link"
+            href="#"
+            >Samsung</a>
```
Comment (anastasiiavorobiova) [inline src/index.html:35]:
```suggestion
            <a 
              class="nav-link"
              href="#"
            >
              Samsung
            </a>
```
Also, consider fixing similar issues further in the code

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Make sure to deploy your page 
Screenshot 2023-11-28 at 13 21 51

**Pull request closed without merging**